### ISSUE TITLE
Wacom tablet detected as game controller

### ISSUE THREAD

**Issue by IrvingBaez** [linuxwacom/libwacom#746]
- **Device name**: Wacom Intuos S
- **Device model identifier**: CTL-4100
- **libwacom version**: 2.10.0-2
- **OS**: Kubuntu 24.04

- [x] I understand that libwacom does affect whether the device works (see [Troubleshooting](https://github.com/linuxwacom/libwacom/wiki/Troubleshooting#my-tablet-doesnt-work))

# Bug description
My Wacom tablet works fine, with pressure sensitivity and all features functioning properly. However, it is being detected as a game controller. As a result, I can't use the GUI to configure it.

![image](https://github.com/user-attachments/assets/a4fb4276-6647-40c5-b50a-5ae237c32bb8)

This is despite being recognized by libwacom:

```
$ libwacom-list-local-devices
devices:
- name: 'Wacom Intuos S'
  bus: 'usb'
  vid: '0x056a'
  pid: '0x0374'
  nodes: 
  - /dev/input/event11: 'Wacom Intuos S Pen'
  - /dev/input/event12: 'Wacom Intuos S Pad'
```

**Comment by IrvingBaez** [comment]
Thanks for pointing out this is not a libwacom related issue. I got confused with the description on the [Project Page](https://linuxwacom.github.io/) since my problem seemed to fit the "control panel does not recognize your tablet and provide appropriate options" case. The issue was actually on Kubuntu.

For anyone having a similar problem: I solved it by installing [wacomtablet](https://github.com/KDE/wacomtablet):

```
sudo add-apt-repository ppa:kubuntu-ppa/backports
sudo apt update
sudo apt install kde-config-tablet
```

And then just finding my tablet with:

`kde_wacom_tabletfinder`
